katello/katello-backup

```diff
@@ -20,7 +20,7 @@ optparse = OptionParser.new do |opts|
     else
       opts.abort("Previous backup directory does not exist: #{dir_path}")
     end
-
+  end
```

Comment:
It'd look better if there was a blank line after this so the two blocks are separated.